robotpos1:
  hardware_interface:
     loop_hz: 10
     joints:
       - joint_1

